.rubocop.yml

```diff
@@ -202,7 +202,7 @@ Style/RedundantReturn:
 Style/IfUnlessModifier:
   Enabled: false
 
-# and and or is okay
+# "and" and "or" are okay
```

Comment:
rule: avoid doubled words.

Here I'm quoting `and` and `or` to avoid breaking that rule, and changing `is` to `are` for verb count agreement.